recipe/meta.yaml

```diff
@@ -15,31 +16,28 @@ build:
 
 requirements:
   host:
-    - python >=3.6
```

Comment:
While 2.6  may be accurate we are only building >=3.6 in conda-forge for a while now. We are in fact about to drop 3.6 soon.

Reply:
Does this mean, the requested package cannot be built?